### PR TITLE
Can't scroll in modal popup on i os

### PR DESCRIPTION
**Pull request by priti2jcommerce**
### Fixed Issues (if relevant)

1. magento/magento2#21147: Can't scroll in modal-popup on iOS
2. ...

### Manual testing scenarios (*)

1. Try to scrolling in the popup on an iPhone (tested on iPhone 6s Plus & iPhone X)
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by priti2jcommerce** [comment]
* @magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by Karlasa** [comment]
@priti2jcommerce 
Thanks for your fix. Your PR basically reverts PR #15172
So could be make both fixes work at the same time :) 
I would suggest to not remove "pointer-events: none;" but just move it under media  min @screen__m so closing modal on clicking outside of popup could work on tablet/desktop. 

Cheers,

**Review by Karlasa — CHANGES_REQUESTED** [review]
@priti2jcommerce
Thanks for your fix. Your PR basically reverts PR #16664
So could be make both fixes work at the same time :)
I would suggest to not remove "pointer-events: none;" but just move it under media min @screen__m so closing modal on clicking outside of popup could work on tablet/desktop.

Cheers,

**Comment by priti2jcommerce** [comment]
Hi @Karlasa 

Thanks for the update, made the changes for it.

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less [hunk 1]
```diff
@@ -152,6 +152,7 @@
 
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
     .modal-popup {
+        pointer-events: auto;
```
Comment (Karlasa) [inline app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less:153]: seems like current solution still not working. It should work but somehow test instance is not generating styles in media check for modal. Have to double check ;)

**Review by Karlasa — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less (hunk 1 above)
Comment (priti2jcommerce) [reply app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less:153]: Hi @Karlasa ,

I have checked and found that in this pr there is current changes not present.

**Comment by Karlasa** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [Magento instance](https://i-21150-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21150-2-3-develop.instances.magento-community.engineering/admin](https://i-21150-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less (hunk 1 above)
Comment (Karlasa) [inline app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less:153]: @priti2jcommerce Could you add extra empty line after that in both files to follow coding standards? :) 

Thanks,

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by Karlasa — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, thank you for the review.
[ENGCOM-4261](https://jira.corp.magento.com/browse/ENGCOM-4261) has been created to process this Pull Request

**Comment by soleksii** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy, here is your [Magento instance](https://i-21150-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21150-2-3-develop.instances.magento-community.engineering/admin](https://i-21150-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by soleksii** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by soleksii** [comment]
Hi @priti2jcommerce ! I've checked PR and it looks like changes doesn't solve the issue #21147
Manual testing scenario:
1. Create simple product in Admin;
2. Create customer in Storefront;
3. Add new address in "Address Book";
4. Add product to Cart;
5. In mini-cart click on "Proceed to Checkout button";
6. Click on "New Address" button;
:x: Can't scroll in modal-popup
Note: Checked on iPhone 6 (iOS v. 12.1.4)
test

**Comment by soleksii** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by priti2jcommerce** [comment]
Hi @stoleksiy ,

I have checked and found that in this pr my changes not reflecting, I have created new one #21616 Please have a look.
Thank you!

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by Karlasa** [comment]
@stoleksiy 
Could you look the CSS file please as i think there are some issues with instances building in test site because looks like this code is missing from CSS file not that the fix is not fixing it :) 

if you add it manually the same way as it should be, it fixes the issue. 

Cheers,

**Comment by soleksii** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy, here is your [new Magento instance](https://pr-21150.instances.magento-community.engineering).
Admin access: [https://pr-21150.instances.magento-community.engineering/admin](https://pr-21150.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=21150) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=21150) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: Karlasa<br/>:white_check_mark: priti2jcommerce<br/>:x: priti<hr/>**priti** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by sidolov** [comment]
Hi @priti2jcommerce looks like you made some commits with email different than in your GitHub profile, please, add email from commits to your profile!
Thank you!

**Comment by priti2jcommerce** [comment]
Hi @sidolov ,

committed changes from GitHub profile are here 
Can you please confirm?
Thanks!

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-4775](https://jira.corp.magento.com/browse/ENGCOM-4775) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed 

**Before:**

before

**After:**

after

**Pull request merged**